_config.yml

```diff
@@ -1,4 +1,4 @@
 theme: jekyll-theme-minimal
 title: Battesimo di Marco
-description: 14 Ottobre 2023 ore 11 Chiesa di San Raimondo Nonnato, Roma 
+description: 14 Ottobre 2023 ore 11 Chiesa di San Raimondo Nonnato, Roma ([qui](https://maps.app.goo.gl/w43DzhpSeExghGmx5))
```

Comment:
forse meglio "clicca qui per aprire l'indirizzo su Google Maps", con il link su "qui"?

Reply:
Il color palette di google.it/maps è disturbantemente diverso da quello google.com/maps, le vie invece di essere gialle sono blue e le aree verdi sono verde smeraldo invece che verde foglia. Tu hai sempre usato la versione it? xD

Reply:
Inoltre credo che visto che sta in yml bisogna mettere in html
<a href="http://example.com">example</a>
see e.g.

https://github.com/jekyll/minima/issues/68

Reply:
@fedlc